rest-api/src/main/java/gov/cms/qpp/conversion/api/controllers/v1/ExceptionHandlerControllerV1.java

```diff
@@ -98,6 +100,16 @@ public ExceptionHandlerControllerV1(final AuditService auditService) {
 
 		return new ResponseEntity<>(exception.getMessage(), httpHeaders, HttpStatus.NOT_FOUND);
 	}
+
+	@ExceptionHandler(AmazonServiceException.class)
+	@ResponseBody
+	ResponseEntity<String> handleAmazonException(AmazonServiceException exception) {
+		API_LOG.error("An S3 error occured", exception);
```

Comment:
Errr, actually this isn't necessarily only an S3 error.  I would say an AWS error occurred.

Reply:
oops forgot to change that, good catch